packages/app-tests/e2e/test-auth-flow.spec.ts

```diff
@@ -10,11 +12,11 @@ test.use({ storageState: { cookies: [], origins: [] } });
 */
 test.describe.configure({ mode: 'serial' });
 
-const username = process.env.E2E_TEST_AUTHENTICATE_USERNAME;
-const email = process.env.E2E_TEST_AUTHENTICATE_USER_EMAIL;
-const password = process.env.E2E_TEST_AUTHENTICATE_USER_PASSWORD;
+const username = 'Test User';
+const email = 'testuser@email.com';
+const password = 'test_password';
 
-test('user can sign up with email and password', async ({ page }: { page: Page }) => {
+test('user can sign up with email and password', async ({ page }) => {
```

Comment:
why remove the types?

Reply:
now the test function is typed, no need to typecast explicitly